BookPlayer/Player/SleepTimer.swift

```diff
@@ -39,6 +40,10 @@ final class SleepTimer {
     1800.0,
     3600.0
   ]
+  
+  public var lastInteral: TimeInterval = 300
```

Comment:
we don't need this property as we already have the `lastActivedState` (just noticed my typo on that var naming 😅), that one is being set in `setTimer(_:)`, but it's being set always, I think we can add a conditional so we don't overwrite it if the user turns off the timer manually

Reply:
damn `lastInteral` typo 😅

Reply:
yeah that's true, I probably abused the sticky state that caused me to use another `last*` variable to actually get the last interval...